composer.json

```diff
@@ -47,7 +47,8 @@
 		"slevomat/coding-standard": "^8.9",
 		"szepeviktor/phpstan-wordpress": "~1.3.0",
 		"wp-graphql/wp-graphql-testcase": "~2.3",
-		"wp-cli/wp-cli-bundle": "^2.8"
+		"wp-cli/wp-cli-bundle": "^2.8",
+		"10up/wp_mock": "^0.5.0"
```

Comment:
Is this PR using this library somewhere?

Reply:
Ah! I was using it when drafting the tests but then decided not to.